src/v2.ts

```diff
@@ -29,7 +29,7 @@ import { DeepPartial } from './cloudevent/types';
  * It will subsequently invoke the cloud function it wraps with the provided {@link CloudEvent}
  */
 export type WrappedV2Function<T extends CloudEvent<unknown>> = (
-  cloudEventPartial?: DeepPartial<T>
+  cloudEventPartial?: DeepPartial<T | object>
```

Comment:
1. I think typing of this should be `DeepPartial<T> |  Object` instead.

2. Since `Object` is roughly similar to `any`, I'm pretty sad to see this function loosing quite a bit of type info. Is there way to make the typing a little more stricter? E.g. `Object` can only have keys that are part of `CloudEvent` type (I'm not sure if my suggestion makes complete sense).

Reply:
1. But `Object` throws a lint error.
2. Thats a rough one. If we want to support raw JSON being passed in for the onValueCreated and onValueDeleted, I'm not sure of any good alternatives.. :\

Reply:
1. TIL.

2. Understood. I wish there were a way to make this type-safe somehow (e.g. after/before example we discussed offline) but I don't see a way.